styles/code.styl

```diff
@@ -133,3 +133,5 @@ div[class~="language-bash"]:before
 
 div[class~="language-php"]:before
   content "php"
+
+@import '~prismjs/themes/prism-' + $prismTheme + '.css'
```

Comment:
Prism's default theme `@import '~prismjs/themes/prism.css` won't be able to configure.

Reply:
Ah, nice catch